src/mame/machine/km035.cpp

```diff
@@ -82,39 +82,39 @@ const tiny_rom_entry *km035_device::device_rom_region() const
 INPUT_PORTS_START( km035 )
 	PORT_START("KBD0")
 	PORT_BIT( 0x01, IP_ACTIVE_HIGH, IPT_UNUSED )
-	PORT_BIT( 0x02, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_NAME("Ctrl") PORT_CODE(KEYCODE_LCONTROL) PORT_CODE(KEYCODE_RCONTROL) PORT_CHAR(UCHAR_SHIFT_2)
+	PORT_BIT( 0x02, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_CODE(KEYCODE_LCONTROL) PORT_CODE(KEYCODE_RCONTROL) PORT_CHAR(UCHAR_SHIFT_2)
```

Comment:
It's a good idea to still provide a `PORT_NAME` for keys like this where the `PORT_CHAR` isn't a printable character.  If you don't, it can mess up the input configuration menu.  Same goes for the other shift/alt/etc. where you've removed the `PORT_NAME`.

Reply:
PORT_NAME is autogenerated for many well-known keys, including these - that's why I removed it.